tests/lang_callable/macros/tfor_ill_formed.nim

```diff
@@ -0,0 +1,19 @@
+discard """
+  description: '''
```

Comment:
this test is testing for statements not macros, it should be in `lang_stmts/for_stmt`